README.md

```diff
@@ -1,239 +1,233 @@
++ **More complicated frontend architectures**: Rounding, localization or other presentational aspects may be performed on the frontend for better interactive performance.
++ **Serverless**: Many Serverless systems use JavaScript as a programming language in order to better leverage the knowledge of frontend engineers.
++ **Server-side programming in JavaScript**: Systems like Node.js and Deno have grown in popularity to do more traditional server-side programming in JavaScript.
 
-#### Goals implied by use case
+In all of these environments, the lack of decimal number support means that various workarounds have to be used (assuming, again, that the programmers are aware of the mismatch between JS’s built-in binary floating-point numbers and proper decimal numbers):
 
-This use case implies the following goals:
-- Avoid unintentional rounding that causes user-visible errors
-- Basic mathematical functions such as `+`, `-`, `*`
-- Sufficient precision for typical money/human-readable quantities
-- Rounding facilities including parameters for both precision and rounding mode
-- Ability to divide a Decimal quantity roughly equally into an integer number of groups
-- Conversion to string, in a locale-sensitive manner
-- Sufficient ergonomics to enable correct usage
-- Be implementable with adequate performance/memory usage for applications
-- Avoid confusing parts of floating point, e.g., -0, NaN, infinity--exceptions make sense for business computing
-- (Please file an issue to mention more requirements)
-
-### Secondary use case: Numerical calculations on more precise floats
++ An external library could be used instead (introducing issues about choosing the library, coordinating on its use). Several options are available:
++ Calculations could be in terms of “cents” (fallible as explained above)
++ In some cases, developers end up using Number instead, believing it to be mostly safe, but in practice causing bugs, even if tries take care of any issues involving rounding or non-exact conversions from decimals to binary floats
 
-If it works out reasonably to provide for it within the same proposal, it would also be nice to provide support for higher-precision applications of floating point numbers.
+#### Goals implied by the main use cases
 
-If Decimal is arbitrary-precision or supports greater precision than Number, it may also be used for applications which need very large floating point numbers, such as astronomical calculations, physics, or even certain games. In some sense, larger or arbitrary-precision binary floats (as supported by [QuickJS](https://bellard.org/quickjs/), or IEEE 754 128-bit/256-bit binary floats) may be more efficient, but Decimal may also be suitable.
+This use case implies the following goals:
 
-#### Sample code
++ Avoid unintentional rounding that causes user-visible errors
++ Basic mathematical functions such as `+`, `-`, `*`, and `/`
++ Sufficient precision for typical money and other human-readable quantities
```

Comment:
Q: does "money" also include crypto stuff where they have a lot of decimal places, or would 5-6 digits be enough to satisfy this goal?